components/LoginForm/LoginForm.tsx

```diff
@@ -5,21 +5,24 @@ import scss from "./LoginForm.module.scss";
 interface LoginFormProps {
     userId: string;
     password: string;
-    setUserId: Function;
-    setPassword: Function;
+    setUserId: (userId: string) => void;
+    setPassword: (password: string) => void;
 }
 
 type FormStatus = "READY" | "FINISHED";
```

Comment:
Do we need this? We should be navigating the user away from this page once login is finished.